Common/Vulkan/VulkanMemory.cpp

```diff
@@ -47,10 +47,16 @@ bool VulkanPushBuffer::AddBuffer() {
 		return false;
 	}
 
+	VkMemoryRequirements reqs;
+	vkGetBufferMemoryRequirements(device_, info.buffer, &reqs);
+
+	// TODO: We really should use memoryTypeIndex here..
+
 	// Okay, that's the buffer. Now let's allocate some memory for it.
 	VkMemoryAllocateInfo alloc = { VK_STRUCTURE_TYPE_MEMORY_ALLOCATE_INFO };
+	// TODO: Should check here that memoryTypeIndex_ matches reqs.memoryTypeBits.
```

Comment:
Any reason not to do it like here?

https://github.com/hrydgard/ppsspp/blob/804aa79376eb4cb216f9f7fa1b8015eff1d3ac59/Common/Vulkan/VulkanMemory.cpp#L162

We could just basically do this later, I think:

https://github.com/hrydgard/ppsspp/blob/804aa79376eb4cb216f9f7fa1b8015eff1d3ac59/Common/Vulkan/VulkanMemory.cpp#L24

-[Unknown]